test/plenario_web/controllers/web/page_controller_test.exs

```diff
@@ -170,6 +170,9 @@ defmodule PlenarioWeb.Web.Testing.PageControllerTest do
   test "explorer receives start datetime greater than end datetime", %{conn: conn} do
     conn = get(conn, page_path(conn, :explorer), %{"starting_on" => "3000-01-01", "ending_on" => "2000-01-01"})
     assert get_flash(conn)["error"] =~ "cannot be greater than"
+
+    conn = get(conn, page_path(conn, :explorer), %{"starting_on" => "2018-03-31", "ending_on" => "2018-05-01"})
+    assert get_flash(conn)["error"] =~ "cannot be greater than"
```

Comment:
I'm guessing you probably want to assert that there _isn't_ a flash, since 2018-03-31 is before 2018-05-01

Reply:
Yep, good catch.